### PR TITLE
Make the output much less verbose in terms of whitespace used...

### PR DESCRIPTION
**Pull request by roesslerj**
- [x] the code follows the [Clean Code](https://clean-code-developer.com/) guidelines.
- [x] the necessary tests are either created or updated.
- [x] all status checks (Travis CI, SonarCloud, etc.) pass.
- [x] your commit history is cleaned up.
- [x] you updated the changelog.
- [ ] you updated necessary documentation within [retest/docs](https://github.com/retest/docs).

## Description

TL;DR: Have a muss shorter error message and general output...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Table of Contents
 
 ### Improvements
 
+* Have the console output much more compact in terms of whitespace used. Con: makes it harder to spot differnce details...
```
Comment (diba1013) [inline CHANGELOG.md]:
```suggestion
* Have the console output much more compact in terms of whitespace used. Con: makes it harder to spot difference details...
```

**Review by diba1013 — CHANGES_REQUESTED** [review]
I would assume that this breaks many tests in recheck-web and recheck.cli. Please prepare corresponding PRs after this has been merged.

**Comment by diba1013** [comment]
For protocol: This reverts #666.

**Review by diba1013 — CHANGES_REQUESTED** [review]
Upon looking at #666, please remove all unnecessary method parameters (mostly `indent`) and simplify method calls wherever possible.

File: src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java [hunk 2]
```diff
@@ -36,29 +36,29 @@ public AttributeDifferencePrinter( final IdentifyingAttributes attributes, final
...
 	private boolean isActualDefault( final AttributeDifference difference ) {
 		return defaultProvider.isDefaultValue( attributes, difference.getKey(), difference.getActual() );
 	}
 
-	private String printActualDefaultDifference( final AttributeDifference difference, final String indent ) {
+	private String printActualDefaultDifference( final AttributeDifference difference ) {
 		final String key = highlighter.highlight( difference.getKey(), HighlightType.KEY );
 		final String expected = highlighter.highlight( difference.getExpectedToString(), HighlightType.VALUE_EXPECTED );
-		final String keyExpectedActualFormat = getKeyExpectedActualFormat( indent );
+		final String keyExpectedActualFormat = getKeyExpectedActualFormat( );
```
Comment (diba1013) [inline src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java]: Is your formatter broken? This should be `()`. (*The same for below*)

**Review by diba1013 — CHANGES_REQUESTED** [review]

File: src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java (hunk 2 above)
Comment (roesslerj) [reply src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java]: I used Eclipse "refactor method signature". Apparently, that doesn't do proper formatting... Done now.

**Review by diba1013 — APPROVED** [review]

**Pull request merged**